airflow/dag/serialization/schema.json

```
@@ -0,0 +1,233 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://airflow.apache.com/schemas/serialized-dags.json",
+  "definitions": {
+    "datetime": {
+      "type": "object",
+      "properties": {
+        "__type": {
+          "type": "string",
+          "const": "datetime"
+        },
+        "__var": {
+          "type": "string",
+          "format": "date-time"
+        }
+      },
+      "required": [
+        "__type",
+        "__var"
+      ],
+      "additionalProperties": false
+    },
+    "timedelta": {
+      "type": "object",
+      "properties": {
+        "__type": {
+          "type": "string",
+          "const": "timedelta"
+        },
+        "__var": {
+          "type": "number",
+          "exclusiveMinimum": 0
```

Just tried out this PR and hit an issue with `airflow db reset`: `jsonschema.exceptions.ValidationError: 0.0 is less than or equal to the minimum of 0` as a result of `timedelta(seconds=0)` in /airflow/tests/dags/test_retry_handling_job.py.

```suggestion
          "minimum": 0
```